data/components/lettuce_async.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "lettuce_async",
+  "type": "client",
+  "name": "lettuce_async",
+  "language": "Java",
+  "label": "Java Async",
+  "repository": {
+      "git_uri": "https://github.com/redis/lettuce"
+  },
+  "examples": {
+      "git_uri": "https://github.com/redis/lettuce",
+      "dev_branch": "doctests",
```

Comment:
Is it possible to have the examples folder in the main branch? We've tried to do that with the other client libs so that errors against the latest build can be spotted quickly.

Reply:
I used the `doctests` branch just to make changes to the docs testable. As soon as [the PR in lettuce](https://github.com/redis/lettuce/pull/3051) is merged, it can be pointed to the main branch.

Reply:
That's excellent, thanks. Great idea :-)